Remotion/Web/ClientScript/Scripts/ListMenu.ts

```diff
@@ -149,65 +159,71 @@ class ListMenu
 
       if (itemInfo.DiagnosticMetadata)
       {
+        // TODO: merge loops?
         // Do not render empty diagnostic metadata attributes
-        $.each(itemInfo.DiagnosticMetadata, function (key, value)
+        for (const [key, value] of Object.entries(itemInfo.DiagnosticMetadata))
         {
           if (value === "" || value === null)
           {
             delete itemInfo.DiagnosticMetadata[key];
           }
-        });
+        }
 
-        $(item).attr(itemInfo.DiagnosticMetadata);
+        for (const [key, value] of Object.entries(itemInfo.DiagnosticMetadata))
+        {
+          item.setAttribute(key, value as string);
```

Comment:
`as string` here is not quite right. Yes `setAttribute` is converting implicitly to a string so it won't matter if the input is a string or not, but in this case it is better to do an explict conversion instead. e.g. `"" + value`